critiquebrainz/frontend/views/moderators.py

```diff
@@ -0,0 +1,25 @@
+from hashlib import md5
+from flask import Blueprint, render_template
+from flask import current_app
+from critiquebrainz.db import users as db_users
+
+moderators_bp = Blueprint('moderators', __name__)
+
+
+@moderators_bp.route('/')
+def moderators():
+    admins = current_app.config['ADMINS']
+    moderators = db_users.get_many_by_mb_username(admins)
+    config_list = [moderator['display_name'] for moderator in moderators]
+    # list of MB users without CritiqueBrainz account
+    missing_users = list(set(admins) - set(config_list))
```

Comment:
https://github.com/metabrainz/critiquebrainz/pull/73#discussion_r95177825. In this function alone you have `admins`, `moderators`, and `users` which all mean exactly the same thing. Perhaps the config variable should be named `MODERATORS` instead of `ADMINS`, but that's out of the scope of this issue. You can just do `moderators = current_app.config['ADMINS']` and be done with it.